aiida_restapi/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+"""pydantic models for AiiDA REST API.
+"""
+# pylint: disable=too-few-public-methods
+
+from .entities import *
```

Comment:
Err, I'm not really a fan of `*` imports, I think explicit is better. Like we've seen with aiida-core, you can end up with sub-classes clobbering namespaces.
I'm open to a discussion on what the best practice is though (is there some PEP that advocates for this?)